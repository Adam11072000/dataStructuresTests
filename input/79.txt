init 6025
quit