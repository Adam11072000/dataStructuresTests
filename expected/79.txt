init done.
addImage: SUCCESS
mergeSuperPixels: FAILURE
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
deleteImage: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: INVALID_INPUT
addImage: FAILURE
mergeSuperPixels: INVALID_INPUT
getHighestScoredLabel: FAILURE
addImage: INVALID_INPUT
mergeSuperPixels: FAILURE
setLabelScore: INVALID_INPUT
quit done.
